src/definitions/modules/modal.less

```diff
@@ -324,12 +328,6 @@
   top: auto !important;
 }
 
-/* Tablet and Mobile */
```

Comment:
Did you test this on mobile to see if everything is good ?

Reply:
I tested everything responsively. This setting is obsolete, that's why I have deleted it. I moved this setting for the close color in basic modals to line 107 instead. The color is the same regardless of the viewport dimensions, so this separate setting didn't make any sense anymore.

Reply:
That's what I wanted to know 😉